### PR TITLE
WIP: MSI: Add aws_ipi and aws_osd workflows

### PR DESCRIPTION
**Pull request by oharan2**

### CODE REVIEW

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-ipi

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-osd

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-ipi

File: ci-operator/step-registry/operator/aws/ipi/operator-aws-ipi-workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
+    - ref: operator-install
+  documentation: |-
+    This workflow provision an OpenShift cluster with default configurations on AWS. 
+    The cluster is set with telemetry step generates a "${SHARED_DIR}/manifest_cluster-monitoring-config.yaml" manifest, and 
+    generates the AWS-specific install-config.yaml contents based on the cluster profile, located in "${SHARED_DIR}/install-config.yaml".
+    It then install required operator on the cluster.
+    After finish testing, the operator will be deleted and cluster will be deprovisioned.
```
Comment (omrirh) [inline ci-operator/step-registry/operator/aws/ipi/operator-aws-ipi-workflow.yaml:19]:
```suggestion
    After finish testing, console logs from AWS instances will be gathered and the cluster will be deprovisioned.
```

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-ipi

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-osd

File: ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-aws-ipi-osd.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      OPENSHIFT_VERSION: "4.13"
+      OPERATOR1_CONFIG: name=rhods-operator;namespace=redhat-ods-operator
+    workflow: operator-aws-ipi
+- as: qetest-operator-aws-osd
+  steps:
+    cluster_profile: aws-managed-cspi-qe
+    env:
+      OPENSHIFT_VERSION: "4.13"
+      OPERATOR1_CONFIG: name=rhods-operator;namespace=redhat-ods-operator
+      REPLICAS: "6"
```
Comment (omrirh) [inline ci-operator/config/CSPI-QE/MSI/CSPI-QE-MSI-rhods-operator__v4.13-aws-ipi-osd.yaml:38]:
```suggestion
      COMPUTE_NODES: "6"
```

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-osd

**Comment by oharan2** [comment]
/pj-rehearse pull-ci-CSPI-QE-MSI-rhods-operator-v4.13-aws-ipi-osd-qetest-operator-aws-osd

**Comment by oharan2** [comment]
Will continue in #41226

**Pull request closed without merging**